packages/react-tinacms-github/src/github-media-store/index.ts

```diff
@@ -50,4 +50,12 @@ export class GithubMediaStore implements MediaStore {
 
     return uploaded
   }
+
+  async previewSrc(src: string) {
+    try {
+      return this.githubClient.getDownloadUrl(src)
+    } catch {
+      return src
```

Comment:
Not sure we should smother this error. I'd say either catch it higher in the call stack or give media store access to the event bus like we're planning with API clients. That doesn't have to be addressed in this PR tho.

In the short term I would prefer to either remove the try-catch or, if we're trying to avoid blowing up someone's whole app, at least returning null / `''`. Returning the original src could cause some unexpected behavior

Reply:
My thought was that you might have an image URL that references a whole URL i.e.

```markdown
![](http://i.ytimg.com/vi/ypG3yBxgDEs/maxresdefault.jpg
```

GithubMediaStore would fail to find that in the repo and just default ot the src

Reply:
The only time this would really fall over is if you are overwriting an existing prod image, `getDownloadUrl` fails, and a root-relative path to the image is returned. User would then see the prod image in the preview, with the appearance that everything is working normally.

I see your point though. IMO, this is actually a significant pitfall when creating a media store. I would be OK with leaving this as-is for now, but it's something to chew on.